MapboxNavigation/RouteMapViewController.swift

```diff
@@ -575,31 +528,6 @@ extension RouteMapViewController: NavigationViewDelegate {
     func didTapInstructionsBanner(_ sender: BaseInstructionsBannerView) {
         self.displayPreviewInstructions()
     }
-    
-    private func displayPreviewInstructions() {
```

Comment:
Did anything change here, or was this simply moved?

Reply:
just moved